pyc_module.cpp

```diff
@@ -130,6 +130,12 @@ void PycModule::setVersion(unsigned int magic)
         m_unicode = true;
         break;
 
+    case MAGIC_352:
```

Comment:
This could be added inline with the `MAGIC_3_5` case above, since the case content is identical.